libnavigation-core/src/test/java/com/mapbox/navigation/core/ContextCreationHelpers.kt

```diff
@@ -0,0 +1,61 @@
+package com.mapbox.navigation.core
+
+import android.app.AlarmManager
+import android.app.NotificationManager
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.content.res.Configuration
+import android.content.res.Resources
+import android.location.LocationManager
+import android.media.AudioManager
+import com.mapbox.android.core.location.LocationEngine
+import com.mapbox.android.core.location.LocationEngineProvider
+import com.mapbox.android.telemetry.MapboxTelemetryConstants.MAPBOX_SHARED_PREFERENCES
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import java.util.Locale
+
+fun createContext(): Context {
```

Comment:
if we'd like to use it across modules, it should be moved to `libtesting` module ⬆️

Reply:
👍